README.md

```diff
@@ -184,6 +187,8 @@ Configuration example:
     $DF Player Damage Modifier: 1.0
     // Enable '/save' and '/load' chat commands (works for all clients) and quick save/load controls handling (works for Dash Faction 1.5.0+ clients). Option designed with run-maps in mind.
     $DF Saving Enabled: false
+    // Randomize the order of levels in the server rotation (instead of loading them sequentially)
+    $DF Randomize Rotation: false
```

Comment:
hm, not sure if I like this name. Maybe "Random Rotation" would be better because it's shorter. Also maybe instead of "Rotation" more explanatory would be "Level Order" or "Level Rotation"?